addons/web/static/src/views/widgets/widget.js

```diff
@@ -34,7 +35,8 @@ export class Widget extends Component {
         return classNames;
     }
     get widgetProps() {
-        const { node: rawNode } = this.props;
+        const { node: rawNode, record } = this.props;
```

Comment:
this could be changed as well: Widget should receive the widgetInfo in props instead of the node. This requires a change in the parsers (change that we did in the model branch), that way, no need for calling `decodeObjectForTemplate`, and it's the exact same API as Field.